demisto_sdk/tests/test_files/fake_pack/.pack-ignore

```diff
@@ -1,5 +1,2 @@
-[demisto-sdk]
-ignore = IN100,IN101
-
 [file:file_name]
-ignore = DO101,BA100
+ignore-BA101
```

Comment:
let's keep it as` ignore = `
and let's support only adding BA101 there - this is for future enhancements